packages/xo-web/src/common/intl/messages.js

```diff
@@ -1258,6 +1258,9 @@ const messages = {
   scheduleCron: 'Cron pattern',
   scheduleName: 'Name',
   scheduleTimezone: 'Timezone',
+  scheduleState: 'State',
+  scheduleStateEnabled: 'Enabled',
+  scheduleStateDisabled: 'Disabled',
```

Comment:
We already had these buttons, how come we need new messages?

Reply:
The messages' name are not generic.